components/docs-chef-io/content/automate/restore.md

```diff
@@ -62,6 +62,15 @@ Ensure access for the backup type used:
          # Use "m" for megabytes and "g" for gigabytes
      ```
 
+{{< note >}}
+
+- If you are restoring the backup taken from an older version of Automate, then you need to provide the `--airgap-bundle </path/to/current/bundle>` with your restore command.
+
+In non-airgapped or Internet-connected environment, user can create airgap-bundle using `chef-automate airgap-bundle create --version <CURRENT_INSTALLED_VERSION>` command.
```

Comment:
Replace it with: - In a non-airgapped or Internet-connected environment; the user can create an airgap-bundle using the `chef-automate airgap-bundle create --version <CURRENT_INSTALLED_VERSION>` command.

Reply:
Done